Update changelog (#1144)
Shopify/polaris-react: CHANGELOG.md

- Fixed examples using the `selected` prop for `Autocomplete` ([#1053](https://github.com/Shopify/polaris-react/pull/1053))

### Development workflow

- Added viewport meta tag to Storybook frame ([#1026](https://github.com/Shopify/polaris-react/pull/1026))

### Code quality

- Removed lodash decorators and replace all occurrences of `_.throttle` with `debounce` ([#1009](https://github.com/Shopify/polaris-react/pull/1009))
- Removed all occurrences of `_.replace` ([#1012](https://github.com/Shopify/polaris-react/pull/1012))
- Added lodash to `create-react-app` example ([#1010](https://github.com/Shopify/polaris-react/pull/1010))
- Updated `create-react-app` example dependencies ([#1010](https://github.com/Shopify/polaris-react/pull/1010))
- Replaced all occurrences of `_.capitalize` with a custom `capitalize` function ([#1015](https://github.com/Shopify/polaris-react/pull/1015))
- Replaced all occurrences of `_.isObject` with a custom `isObject` function ([#1011](https://github.com/Shopify/polaris-react/pull/1011))
- Replaced all occurrences of `_.get` with a custom `get` function ([#1013](https://github.com/Shopify/polaris-react/pull/1013))

> - Moved icons specific to Banner, DropZone, and ResourceList to polaris-icons ([#1042](https://github.com/Shopify/polaris-react/pull/1042))

Reviewer: Adding a link could be useful for partners, as they don't have access to other polaris-icons project pages:

```suggestion
- Moved icons specific to `Banner`, `DropZone`, and `ResourceList` to [@shopify/polaris-icons](https://www.npmjs.com/package/@shopify/polaris-icons) ([#1042](https://github.com/Shopify/polaris-react/pull/1042))
```

> - Spinner now uses the Image component instead of an SVG tag to render ([#1042](https://github.com/Shopify/polaris-react/pull/1042))

Reviewer: ```suggestion
- Spinner now uses the `Image` component instead of an SVG tag to render ([#1042](https://github.com/Shopify/polaris-react/pull/1042))
```

### Deprecations

> - Passing a React Element into the Icon component is now deprecated in favor of passing a React Component ([#1042](https://github.com/Shopify/polaris-react/pull/1042))

Reviewer: ```suggestion
- Passing a React Element into the `Icon` component is now deprecated in favor of passing a React Component ([#1042](https://github.com/Shopify/polaris-react/pull/1042))
```

> - Passing a React Element into the Icon component is now deprecated in favor of passing a React Component ([#1042](https://github.com/Shopify/polaris-react/pull/1042))

Reviewer 2: ```suggestion
- Deprecated passing a React Element into the `Icon` component in favor of passing a React Component ([#1042](https://github.com/Shopify/polaris-react/pull/1042))
```

> - The untrusted prop in the Icon component is now deprecated ([#1042](https://github.com/Shopify/polaris-react/pull/1042))

Reviewer 2: ```suggestion
- Deprecated the untrusted prop in the `Icon` component ([#1042](https://github.com/Shopify/polaris-react/pull/1042))
```

### Development workflow

- Added viewport meta tag to Storybook frame ([#1026](https://github.com/Shopify/polaris-react/pull/1026))

### Code quality

- Removed lodash decorators and replace all occurrences of `_.throttle` with `debounce` ([#1009](https://github.com/Shopify/polaris-react/pull/1009))
- Removed all occurrences of `_.replace` ([#1012](https://github.com/Shopify/polaris-react/pull/1012))
- Add lodash to `create-react-app` example ([#1010](https://github.com/Shopify/polaris-react/pull/1010))
- Update `create-react-app` example dependencies ([#1010](https://github.com/Shopify/polaris-react/pull/1010))
- Replace all occurrences of `_.capitalize` with a custom `capitalize` function ([#1015](https://github.com/Shopify/polaris-react/pull/1015))
- Replace all occurrences of `_.isObject` with a custom `isObject` function ([#1011](https://github.com/Shopify/polaris-react/pull/1011))
- Replace all occurrences of `_.get` with a custom `get` function ([#1013](https://github.com/Shopify/polaris-react/pull/1013))
- Banner, DropZone, and ResourceList’s icons were moved to polaris-icons ([#1042](https://github.com/Shopify/polaris-react/pull/1042))

> - Spinner now uses the Image component instead of an SVG tag to render ([#1042](https://github.com/Shopify/polaris-react/pull/1042))

Reviewer 2: ```suggestion
- Updated spinner component to use the Image component instead of an SVG tag to render ([#1042](https://github.com/Shopify/polaris-react/pull/1042))
```

---

## 3.10.0 - 2019-03-07

### Enhancements

> - Track Polaris version information in App Bridge actions ([#1087](https://github.com/Shopify/polaris-react/pull/1087))

Reviewer 2: This should be rephrased as past tense:

```suggestion
- Added Polaris version information tracking in App Bridge actions ([#1087](https://github.com/Shopify/polaris-react/pull/1087))
```

- Re-added the navigation’s border-right ([#1096](https://github.com/Shopify/polaris-react/pull/1096))
- Added `onScrolledToBottom` prop to `Modal` ([#1117](https://github.com/Shopify/polaris-react/pull/1117))
- Updated `Navigation.Item` to use `Icon` when `iconBody` prop is passed in. Renders these icons in an `img` tag now. ([#1094](https://github.com/Shopify/polaris-react/pull/1094))
- Added focus state outlines to be visible when using Windows High Contrast Mode for `Button` ([#1101](https://github.com/Shopify/polaris-react/pull/1101))

### Bug fixes

> - Revert a change that constrained `DropZone` height based on inherited wrapper height [#1129](https://github.com/Shopify/polaris-react/pull/1129)

Reviewer 2: ```suggestion
- Reverted a change that constrained `DropZone` height based on inherited wrapper height [#1129](https://github.com/Shopify/polaris-react/pull/1129)
```

> - Revert a change that constrained `DropZone` height based on inherited wrapper height [#1129](https://github.com/Shopify/polaris-react/pull/1129)

Reviewer: ```suggestion
- Reverted a change that constrained `DropZone` height based on inherited wrapper height [#1129](https://github.com/Shopify/polaris-react/pull/1129)
```

- Fixed missing rounded corners on `Tag` button states ([#1078](https://github.com/Shopify/polaris-react/pull/1078))
- Removed reference to `window.Polaris`, which in some cases could be undefined ([#1104](https://github.com/Shopify/polaris-react/issues/1104))
- Added padding and margin to `subdued` sections for proper spacing between the header and footer ([#1082](https://github.com/Shopify/polaris-react/pull/1082))
- Removed left margin from vertical `Stack` to prevent overflow ([#1024](https://github.com/Shopify/polaris-react/pull/1024))
- Fixed the size differences between `SkeletonThumbnail` and `Thumbnail` ([#1141](https://github.com/Shopify/polaris-react/pull/1141) (thanks [@mbaumbach](https://github.com/mbaumbach) for the [issue report](https://github.com/Shopify/polaris-react/issues/1135)))
- Refactored `ComboBox` tests that were not running ([#1137](https://github.com/Shopify/polaris-react/pull/1137))

### Documentation

> - Updated related component documentation for `Page`, `Page actions`, and `Pagination` ([#1103](https://github.com/Shopify/polaris-react/pull/1103))

Reviewer: ```suggestion
- Updated related component documentation for `Page`, `PageActions`, and `Pagination` ([#1103](https://github.com/Shopify/polaris-react/pull/1103))
```

> - Made documentation for properties only available in a stand-alone context more explicit for `Modal` ([#1065](https://github.com/Shopify/polaris-react/pull/1065))

Reviewer 2: ```suggestion
- Improved `Modal` documentation for properties only available in a stand-alone app context ([#1065](https://github.com/Shopify/polaris-react/pull/1065))
```

- Added accessibility documentation about `Banner` ([#1071](https://github.com/Shopify/polaris-react/pull/1071))
- Added accessibility documentation for `InlineError` ([#1073](https://github.com/Shopify/polaris-react/pull/1073))
- Added accessibility documentation for `Loading`([#1075](https://github.com/Shopify/polaris-react/pull/1075))
- Fixed documentation about the `ariaPressed` prop for `Button` ([#1097](https://github.com/Shopify/polaris-react/pull/1097))
- Fixed examples using the `selected` prop for `Autocomplete` ([#1053](https://github.com/Shopify/polaris-react/pull/1053))

### Development workflow

- Added viewport meta tag to Storybook frame ([#1026](https://github.com/Shopify/polaris-react/pull/1026))

> ### Dependency upgrades

Reviewer: ```suggestion
```

### Code quality

- Removed lodash decorators and replace all occurrences of `_.throttle` with `debounce` ([#1009](https://github.com/Shopify/polaris-react/pull/1009))
- Removed all occurrences of `_.replace` ([#1012](https://github.com/Shopify/polaris-react/pull/1012))

> - Add lodash to `create-react-app` example ([#1010](https://github.com/Shopify/polaris-react/pull/1010))

Reviewer: ```suggestion
- Added lodash to `create-react-app` example ([#1010](https://github.com/Shopify/polaris-react/pull/1010))
```

> - Add lodash to `create-react-app` example ([#1010](https://github.com/Shopify/polaris-react/pull/1010))

Reviewer 2: ```suggestion
- Added lodash to `create-react-app` example ([#1010](https://github.com/Shopify/polaris-react/pull/1010))
```

> - Update `create-react-app` example dependencies ([#1010](https://github.com/Shopify/polaris-react/pull/1010))

Reviewer: ```suggestion
- Updated `create-react-app` example dependencies ([#1010](https://github.com/Shopify/polaris-react/pull/1010))
```

> - Update `create-react-app` example dependencies ([#1010](https://github.com/Shopify/polaris-react/pull/1010))

Reviewer 2: ```suggestion
- Updated `create-react-app` example dependencies ([#1010](https://github.com/Shopify/polaris-react/pull/1010))
```

> - Replace all occurrences of `_.capitalize` with a custom `capitalize` function ([#1015](https://github.com/Shopify/polaris-react/pull/1015))

Reviewer 2: ```suggestion
- Replaced all occurrences of `_.capitalize` with a custom `capitalize` function ([#1015](https://github.com/Shopify/polaris-react/pull/1015))
```

> - Replace all occurrences of `_.capitalize` with a custom `capitalize` function ([#1015](https://github.com/Shopify/polaris-react/pull/1015))

Reviewer: ```suggestion
- Replaced all occurrences of `_.capitalize` with a custom `capitalize` function ([#1015](https://github.com/Shopify/polaris-react/pull/1015))
```

> - Replace all occurrences of `_.isObject` with a custom `isObject` function ([#1011](https://github.com/Shopify/polaris-react/pull/1011))

Reviewer 2: ```suggestion
- Replaced all occurrences of `_.isObject` with a custom `isObject` function ([#1011](https://github.com/Shopify/polaris-react/pull/1011))
```

> - Replace all occurrences of `_.isObject` with a custom `isObject` function ([#1011](https://github.com/Shopify/polaris-react/pull/1011))

Reviewer: ```suggestion
- Replaced all occurrences of `_.isObject` with a custom `isObject` function ([#1011](https://github.com/Shopify/polaris-react/pull/1011))
```

> - Replace all occurrences of `_.get` with a custom `get` function ([#1013](https://github.com/Shopify/polaris-react/pull/1013))

Reviewer: ```suggestion
- Replaced all occurrences of `_.get` with a custom `get` function ([#1013](https://github.com/Shopify/polaris-react/pull/1013))
```

> - Replace all occurrences of `_.get` with a custom `get` function ([#1013](https://github.com/Shopify/polaris-react/pull/1013))

Reviewer 2: ```suggestion
- Replaced all occurrences of `_.get` with a custom `get` function ([#1013](https://github.com/Shopify/polaris-react/pull/1013))
```

> - Banner, DropZone, and ResourceList’s icons were moved to polaris-icons ([#1042](https://github.com/Shopify/polaris-react/pull/1042))

Reviewer 2: ```suggestion
- Moved icons specific to Banner, DropZone, and ResourceList to polaris-icons ([#1042](https://github.com/Shopify/polaris-react/pull/1042))
```
